testing/tests/DevExpress.ui.widgets/pager.tests.js

```diff
@@ -142,69 +167,100 @@ QUnit.module('Pager', {
 
-        assert.ok(this.checkPages(instance._pages, [1, 2, 3, 4, 5, 13, '>'], '1'), 'pages');
+        const instance = $pager.dxPager('instance');
+
+        if(isRenovatedComponent) {
+            assert.strictEqual($pager.find('.dx-next-button').length, 1, 'pager has next page button');
+            assert.strictEqual($pager.find('.dx-prev-button').length, 0, 'pager doesnt have prev page button');
+            assert.ok(this.checkPages(instance._pages, [1, 2, 3, 4, 5, 13], '1'), 'pages');
+        } else {
+            assert.ok(this.checkPages(instance._pages, [1, 2, 3, 4, 5, 13, '>'], '1'), 'pages');
+        }
     });
 
     // B232538
     QUnit.test('Get pages when pages count more maxPagesCount after pages count is changed', function(assert) {
-        const $pager = $('#container').dxPager({ maxPagesCount: 10, pageCount: 1 }); const instance = $pager.dxPager('instance');
+        const $pager = $('#container').dxPager({ maxPagesCount: 10, pageCount: 1 });
+        const instance = $pager.dxPager('instance');
 
         instance.option('pageCount', 13);
 
         assert.ok(this.checkPages(instance._pages, [1, 2, 3, 4, 5, 13], '1'), 'pages');
     });
 
     QUnit.test('Get pages when more page does not exist after changed', function(assert) {
-        const $pager = $('#container').dxPager({ maxPagesCount: 10, pageCount: 13, hasKnownLastPage: false }); const instance = $pager.dxPager('instance');
+        const $pager = $('#container').dxPager({ maxPagesCount: 10, pageCount: 13, hasKnownLastPage: false });
+        const instance = $pager.dxPager('instance');
 
         instance.option({ pageCount: 14, hasKnownLastPage: true });
 
         assert.ok(this.checkPages(instance._pages, [1, 2, 3, 4, 5, 14], '1'), 'pages');
     });
 
     QUnit.test('Get pages when pages count less maxPagesCount', function(assert) {
-        const $pager = $('#container').dxPager({ maxPagesCount: 10, pageCount: 7 }); const instance = $pager.dxPager('instance');
-
+        const $pager = $('#container').dxPager({ maxPagesCount: 10, pageCount: 7 });
+        const instance = $pager.dxPager('instance');
         assert.ok(this.checkPages(instance._pages, [1, 2, 3, 4, 5, 6, 7], '1'), 'pages');
     });
 
     QUnit.test('SelectPageByValue', function(assert) {
-        const $pager = $('#container').dxPager({ maxPagesCount: 10, pageCount: 13 }); const instance = $pager.dxPager('instance'); const pages = instance._pages;
+        const $pager = $('#container').dxPager({ maxPagesCount: 10, pageCount: 13 });
+        const instance = $pager.dxPager('instance');
+        const pages = () => instance._pages;
 
         // assert
-        assert.ok(this.checkPages(pages, [1, 2, 3, 4, 5, 13], '1'), 'page value = 1');
+        assert.ok(this.checkPages(pages(), [1, 2, 3, 4, 5, 13], '1'), 'page value = 1');
+        instance._useDefaultOptionUpdate = true;
+        $(instance._pages[1]._$page).trigger('dxclick');
```

Comment:
Why is trigger added? The next step with `instance._selectPageByValue(2);` has the same result state of pages.